hudi-utilities/src/main/java/org/apache/hudi/utilities/deltastreamer/HoodieDeltaStreamer.java

```diff
@@ -137,9 +138,9 @@ public HoodieDeltaStreamer(Config cfg, JavaSparkContext jssc, FileSystem fs, Con
 
     this.cfg = cfg;
     this.bootstrapExecutor = Option.ofNullable(
```

Comment:
Can we instantiate JavaSparkContext as local variable and use it in lines 141 and 143.

Reply:
> Can we instantiate JavaSparkContext as local variable and use it in lines 141 and 143.

done